workflow/controller/controller.go

```diff
@@ -763,7 +763,7 @@ func (wfc *WorkflowController) newPodInformer() cache.SharedIndexInformer {
 // The defaults for the workflow controller are set in the workflow-controller config map
 func (wfc *WorkflowController) setWorkflowDefaults(wf *wfv1.Workflow) error {
```

Comment:
do we have enough tests for this func? are we 100% sure behaviour is unchanged?

Reply:
yes, This is an existing function with unit test.  All tests are passed. I reviewed the unit tests.